internal/route/repo/pull.go

```diff
@@ -39,6 +39,14 @@ var (
 	}
 )
 
+var (
+	PullRequestTitleTemplateCandidates = []string{
```

Comment:
I think this can be moved to after line 39, we don't need separate `var` groups, especially these two variables are same kind.

Reply:
thnx for review/attention.
it fixed